.github/workflows/release-please.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  push:
+    branches:
+      - master
+
+permissions:
+  contents: write
+  pull-requests: write
+
+name: release-please
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: googleapis/release-please-action@v4
+        with:
+          # this assumes that you have created a personal access token
+          # (PAT) and configured it as a GitHub action secret named
+          # `MY_RELEASE_PLEASE_TOKEN` (this secret name is not important).
+          token: ${{ secrets.GH_PAT }}
+          # this is a built-in strategy in release-please, see "Action Inputs"
+          # for more options
+          config-file: release-please-config.json
```

Comment:
This can be removed, it is the default file naming.

Reply:
Done